modules/ROOT/pages/tutorials/aura-query-api-tutorial.adoc

```diff
@@ -0,0 +1,79 @@
+[[aura-query-api-tutorial]]
+= Aura Query API
+:description: This page describes how to create a node using Query API and an Aura instance.
+
+== Introduction
+
+The Query API enables interaction with an Aura database using Cypher via HTTPS.
+
+This tutorial guides you through the process of sending Cypher queries to an Aura instance using the Query API.
+
+For more information, see the link:https://neo4j.com/docs/query-api/current/[Query API documentation] but note that:
+
+* For Aura instances, the host follows the format `<databaseID>.databases.neo4j.io`
+
+* The official documentation mainly refers to self-managed instances that use port `:7474` however Aura supports HTTPS, which defaults to port `:443` 
+
+* As HTTPS implies port `:443` in the request, you do not need to include the port in the request.
```

Comment:
```suggestion
* As HTTPS implies port `443` in the request, you do not need to include the port in the URL.
```

The request is the blob of data (headers+body) that leaves your machine to reach the Neo4j server. 
The URL is the location of the server (made up of, in order: scheme (http/https), host (imbeautiful.neo4j.io), port (443), path (/query/v2/), querystring (?key=value&another=more)).